src/pages/reader/epub.vue

```diff
@@ -347,6 +378,17 @@ export default {
         );
       }
     },
+    adjustFontSize(indexDiff) {
+      const currentIndex = FONT_SIZES.indexOf(this.fontSize);
```

Comment:
Storing fontSizeIndex as data, bound it by math.max()/min() seems more straight forward

Reply:
Updated